src/components/Tabs/Tabs.tsx

```diff
@@ -186,7 +186,7 @@ export const Tabs = chakra(
 
       const previousButton = (
         <Button
-          aria-label="Previous"
+          aria-label="Scroll tabs left"
```

Comment:
Looking at this update, I feel we should also add a more descriptive `title` prop on the `Icon` components.  Can you also add `title="left arrow"` and `title="right arrow"` to the respective `Icon` components?